src/components/pages/pricing/calculator/calculator.jsx

```diff
@@ -106,190 +109,157 @@ const Calculator = () => {
   );
 
   return (
-    <section className="faq safe-paddings my-40 2xl:my-32 xl:my-28 lg:my-24 md:my-20">
+    <section className="faq safe-paddings mb-40 mt-[17.25rem] 2xl:my-32 xl:my-28 lg:my-24 md:my-20">
       <Container size="mdDoc">
         <div className="mx-auto flex max-w-[972px] flex-col items-center">
-          <span className="text-center text-lg uppercase leading-snug text-primary-1">
-            Pricing Calculator
-          </span>
-          <h2 className="mt-2.5 inline-flex flex-col text-center text-5xl font-bold leading-tight 2xl:max-w-[968px] 2xl:text-[44px] 2xl:leading-snug xl:text-4xl lg:inline lg:text-[36px] lg:leading-tight">
-            Calculate your monthly bill based <br /> on compute time and storage
-          </h2>
+          <Heading className="text-center lg:inline" badge="Pricing Calculator" tag="h2" size="2sm">
+            Calculate your monthly bill based <br className="sm:hidden" /> on compute time and
```

Comment:
Just a recommendation - you could use `<wbr />` tag to set a possible place to break a string. It's breaking only when it needs to, so there would be no need to write `sm:hidden`.

Reply:
Thanks, good catch!